fe/fe-core/src/main/java/org/apache/doris/load/routineload/KafkaRoutineLoadJob.java

```diff
@@ -741,7 +752,8 @@ public boolean hasMoreDataToConsume(UUID taskId, Map<Integer, Long> partitionIdT
             // all offsets to be consumed are newer than offsets in cachedPartitionWithLatestOffsets,
             // maybe the cached offset is out-of-date, fetch from kafka server again
             List<Pair<Integer, Long>> tmp = KafkaUtil.getLatestOffsets(id, taskId, getBrokerList(),
-                    getTopic(), getConvertedCustomProperties(), Lists.newArrayList(partitionIdToOffset.keySet()));
+                    getTopic(), getConvertedCustomProperties(null),
```

Comment:
"hasMoreDataToConsume" will send RPC to Kafka , I think it still need corresponding rack as param.